tests/ctst/world/Zenko.ts

```diff
@@ -114,6 +114,8 @@ export default class Zenko extends World<ZenkoWorldParameters> {
     static readonly PRIMARY_SITE_NAME = 'admin';
     static readonly SECONDARY_SITE_NAME = 'dradmin';
 
+    public praInstallCount = 0;
```

Comment:
For the future in case we have other scenarios, we can set it in the cache of CTST instead...